style.css

```diff
@@ -417,3 +344,506 @@ footer h4 {
   margin: 0 20px 40px 19px;
   padding-left: 0;
 }
+
+/* hovers and active */
+
+.start:hover {
+  background-color: #fff;
+  transition: 0.7ms;
+  color: #1a2236;
+  border: solid 2px #1a2236;
+}
+
+.start:active {
+  background-color: #fff;
+  transition: 0.7ms;
+  color: #1a2236;
+  border: solid #1a2236;
+}
+
+.see:hover {
+  background-color: #1a2236;
+  transition: 0.7ms;
+  color: #fff;
+  border: solid 2px #fff;
+}
+
+.see:active {
+  background-color: #1a2236;
+  transition: 0.7ms;
+  color: #fff;
+  border: solid #fff;
+}
+
+.cvBut:hover {
+  background-color: #fff;
+  transition: 0.7ms;
+  color: #1a2236;
+  border: solid 2px #1a2236;
+}
+
+.cvBut:active {
+  background-color: #fff;
+  transition: 0.7ms;
+  color: #1a2236;
+  border: solid #1a2236;
+}
+
+.getBut:hover {
+  background-color: #fff;
+  transition: 0.7ms;
+  color: #1a2236;
+  border: solid 2px #1a2236;
+}
+
+.getBut:active {
+  background-color: #fff;
+  transition: 0.7ms;
+  color: #1a2236;
+  border: solid #1a2236;
+}
+
+.cvBut img:last-child {
+  display: none;
+}
+
+.see img:last-child {
+  display: none;
+}
+
+.cvBut:hover img:last-child {
+  display: block;
+}
+
+.see:hover img:last-child {
+  display: block;
+}
+
+.cvBut:hover img:first-child {
+  display: none;
+}
+
+.see:hover img:first-child {
+  display: none;
+}
+
+/* Media Query */
+@media screen and (min-width: 768px) {
+  body {
+    max-width: 100vw;
+  }
+
+  /* header */
+  .headbar1 {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: white;
+    height: 10vh;
+    width: 100%;
+  }
+
+  .nav-list {
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+    padding: 0;
+    list-style-type: none;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+  }
+
+  a {
+    text-decoration: none;
+    color: #1a2236;
+  }
+
+  /* headline */
+  .pic1 {
+    display: none;
+  }
+
+  .img-container {
+    width: 50%;
+    height: 90vh;
+  }
+
+  .img-back-big {
+    width: 100%;
+    height: 90vh;
+    background-image: url(./images/big-background.png);
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+  }
```

Comment:
- [ ] (Optional) I would like to advise you to remove this CSS declaration as it makes the background image look squeezed. 👍